### DISCUSSION TITLE
How to Configure Separate Deployments for 1 Repo with 2 projects

### DISCUSSION THREAD

**Discussion by LowScarlet** [vercel/community#7283] [Help]
### Summary

How can I ensure that only the frontend1 folder is deployed when there are changes in that folder, without redeploying the frontend2 folder which has no changes, for my project with the structure:

- root
---- frontend1
---- frontend2

that has been deployed per folder?

### Steps to Reproduce

whenever I make changes to the frontend1 folder and push these changes to the repository, both frontend1 and frontend2 get redeployed. This is inefficient and unnecessary because there are no changes in the frontend2 folder.

I need a way to ensure that only the frontend1 folder gets deployed when changes are made to it, without triggering a redeployment for the frontend2 folder if it remains unchanged.

**Comment by versecafe** [comment]
Vercel will create a deploy per commit, you could use something like git submodules potentially or let it redeploy anyways and use [turborepo](https://turbo.build/) which will use a cache if nothing is changed making build times and deploy times as low as 30 seconds at which point you don't really need to worry about it.

[a comment by an automated account is not shown]
